WordPressShared/Core/Utility/NSDate+Helpers.swift

```diff
@@ -120,19 +108,7 @@ extension Date {
 
     @available(*, deprecated, renamed: "toMediumString", message: "Removed to help drop the deprecated `FormatterKit` dependency – @jkmassel, Mar 2021")
     public func mediumString(timeZone: TimeZone? = nil) -> String {
-        let relativeFormatter = TTTTimeIntervalFormatter()
-        let absoluteFormatter = DateFormatters.mediumDate
-
-        if let timeZone = timeZone {
-            absoluteFormatter.timeZone = timeZone
-        }
-
-        let components = Calendar.current.dateComponents([.day], from: self, to: Date())
-        if let days = components.day, abs(days) < 7 {
-            return relativeFormatter.string(forTimeInterval: timeIntervalSinceNow)
-        } else {
-            return absoluteFormatter.string(from: self)
-        }
+        toMediumString(inTimeZone: timeZone)
```

Comment:
What do you think about removing this method altogether?

Given this is a `public` API, doing so would be a breaking change and require a new major version, but I think it's okay since we at Automattic are likely to only active consumers of this library, and the upgrade cost for us is very low.

Reply:
Sounds reasonable to me. I remember there are a couple other deprecated methods, which can probably go away all together.